frontend/public/components/modals/create-namespace-modal.jsx

```diff
@@ -114,6 +116,23 @@ const CreateNamespaceModalWithTranslation = connect(
         [allow]: t('public~No restrictions'),
         [deny]: t('public~Deny all inbound traffic'),
       };
+
+      const popOverText = () => {
+        const type = this.props.createProject ? t('public~project') : t('public~namespace');
+        const nameFormat = t(
+          "public~A {{type}} name must consist of lower case alphanumeric characters or '-', and must start and end with an alphanumeric character (e.g. 'my-name' or '123-abc').",
+          { type },
+        );
+        const createNameSpace = t(
```

Comment:
Naming nit: namespace is one word, so this would be `createNamespace`

Reply:
Changed